site/content/features/typepatterns.md

```diff
@@ -492,11 +472,11 @@ case String s when s.length() > 3: ... // No break/yield required
 default: ...
 ```
 
-You cannot fall through a type pattern. That is, you cannot fall into the the second case. After all, falling through skips the test and goes directly into the code that follows. However, you can fall through from a type pattern into another case that isn't a type pattern. In the preceding example, it is ok to fall through the default.
+You cannot fall through a type pattern with a variable binding. That is, you cannot fall into the the second case. After all, falling through skips the test and goes directly into the code that follows. However, you can fall through from a type pattern into another case that isn't a type pattern, or that is a type pattern with an unnamed variable. In the preceding example, it is ok to fall through the default.
 
 Here is a complete example, as contrived as all fall through examples that I have ever seen.
 
-{{< sandbox version=java19 preview="true" mainclass="Main" >}}{{< sandboxsource "Main.java" >}}
+{{< sandbox version=java21 preview="true" mainclass="Main" >}}{{< sandboxsource "Main.java" >}}
```

Comment:
Is this sandbox supposed to run without errors? I get the following compiler error:

    Main.java:20: error: this case label is dominated by a preceding case label
             case String s: yield '"' + s + '"';

Also `preview` not required any more.

Reply:
In Java 21 default dominates. I fixed the code.